web/themes/custom/custom_theme/custom_theme.theme

```diff
@@ -9,7 +9,7 @@
  */
 function custom_theme_preprocess_views_view(&$variables) {
   // Check if the view id is of required type or not.
-  if($variables['id'] === 'movies' && $variables['display_id'] === 'page_1') {
+  if ($variables['id'] === 'movies' && $variables['display_id'] === 'page_1') {
     $variables['#attached']['library'][] = 'custom_theme/movie-slider';
   }
 }
```

Comment:
check if we can target view directly by hook name and if  there is no need to  use if condition